src/riakc_ts_query_operator.erl

```diff
@@ -31,9 +31,12 @@
 -export([serialize/2,
          deserialize/1, deserialize/2]).
 
-serialize(QueryText, Interpolations) ->
+
+serialize(QueryText, Interpolations)
+  when is_binary(QueryText) orelse is_list(QueryText) ->
+    Q = unicode:characters_to_binary(QueryText),
```

Comment:
```erlang
Erlang R16B02_basho10 (erts-5.10.3) [source] [64-bit] [smp:2:2] [async-threads:10] [kernel-poll:false] [frame-pointer]

Eshell V5.10.3  (abort with ^G)
1> unicode:characters_to_binary(<<"\xc3\xb1">>).
<<"ñ"/utf8>>
2> unicode:characters_to_binary(<<"abcde">>).
<<"abcde">>
3> unicode:characters_to_binary(<<"\xa0\xa1">>).
{error,<<>>,<<" ¡">>}
4> unicode:characters_to_binary(<<"\xf0\x28\x8c\x28">>).
{error,<<>>,<<240,40,140,40>>}
5> unicode:characters_to_binary(<<"фафа">>).
<<"D0D0">>
```

@hmmr - your concern is that the user will receive the `error` tuple but may not know why?